### PR TITLE
Add time measure for `AccountStateDeltaImpl.GetState()` method

### PR DESCRIPTION
**Pull request by limebell**

### CODE REVIEW

File: Libplanet/Action/AccountStateDeltaImpl.cs [hunk 1]
```diff
@@ -94,10 +95,22 @@ Address signer
...
+        IValue? IAccountStateView.GetState(Address address)
+        {
+            var stopwatch = new Stopwatch();
+            stopwatch.Start();
+            var states = UpdatedStates.TryGetValue(address, out IValue? value)
                 ? value
                 : StateGetter(new[] { address })[0];
+            Log.Logger
+                .ForContext("Tag", "Metric")
+                .ForContext("Subtag", "LoadActionsDuration")
```
Comment (greymistcube) [inline Libplanet/Action/AccountStateDeltaImpl.cs:107]: Maybe `LoadStateDuration` might be more apt? 😶

File: Libplanet/Action/AccountStateDeltaImpl.cs [hunk 2]
```diff
@@ -94,10 +95,22 @@ Address signer
...
+                .Information(
+                    "Took {DurationMs} ms to get state {Address}",
```
[the remark this answers is not in the document]
Comment (greymistcube) [reply Libplanet/Action/AccountStateDeltaImpl.cs:109]: Also maybe we can also log `states.EncodedLength`? 😗

[the remark this answers is not in the document]
Comment (greymistcube) [reply Libplanet/Action/AccountStateDeltaImpl.cs:109]: Aaaaand also also maybe `states.Kind`, if there is no particular concern about performance.

[the remark this answers is not in the document]
Comment (limebell) [reply Libplanet/Action/AccountStateDeltaImpl.cs]: Amended! Please check https://github.com/planetarium/libplanet/pull/2979/commits/a9c34db11dadc8734faf0899c05f33ddc7dc2741

**Pull request merged**